src/main/java/org/assertj/core/api/AbstractIterableAssert.java

```diff
@@ -924,6 +925,45 @@ public S usingDefaultElementComparator() {
 	return new ListAssert<>(result);
   }
 
+    /**
+     * Extract the Iterable values from Iterable's elements under test by applying an Iterable extracting function based
+     * a property or fieldname and concatenating the result lists. The returned iterable becomes a new object under test.
+     * <p/>
+     * It allows testing the results of extracting values that are represented by Iterables.
+     * <p/>
+     * For example:
+     *
+     * <pre><code class='java'>
+     * List&lt;CartoonCharacter&gt; cartoonCharacters = newArrayList();
+     *
+     * CartoonCharacter bart = new CartoonCharacter("Bart Simpson");
+     * CartoonCharacter lisa = new CartoonCharacter("Lisa Simpson");
+     * CartoonCharacter maggie = new CartoonCharacter("Maggie Simpson");
+     *
+     * CartoonCharacter homer = new CartoonCharacter("Homer Simpson");
+     * homer.getChildren().add(bart);
+     * homer.getChildren().add(lisa);
+     * homer.getChildren().add(maggie);
+     *
+     * CartoonCharacter pebbles = new CartoonCharacter("Pebbles Flintstone");
+     * CartoonCharacter fred = new CartoonCharacter("Fred Flintstone");
+     * fred.getChildren().add(pebbles);
+     *
+     *
+     * assertThat(Lists.newArrayList(homer, fred)).flatExtracting("children").containsOnly(bart, lisa, maggie, pebbles);
```

Comment:
get rid of `Lists.` as we use `newArrayList` directly when creating `cartoonCharacters`, it will make the example lighter.
It would also be good to put `.containsOnly` in the next line, aligned on `.flatExtracting`